docs/DeveloperGuide.md

```diff
@@ -6,33 +6,230 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### The Overall UML Diagram:
 
+![UML Diagram](assets/images/mathGenius.png)    
+
+
+### The overall Sequence Diagram:
+![Sequence Diagram](assets/images/sequenceDiagram.png)    
```

Comment:
![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/110598041/b277e220-5921-453a-8e42-bafbdfab2872)
Is the activation bar is supposed to end mid sequence for duke? or is it supposed to continue all the way down